content/_changelogs/11.1.0.md

```diff
@@ -0,0 +1,30 @@
+## 11.1.0
+
+_Released 11/14/2022_
+
+**Features:**
+
+- Next.js v13 is now supported.
+  [#24396](https://github.com/cypress-io/cypress/pull/24396)
+
+**Bugfixes:**
+
+- Fixed an 11.0.0 regression where Typescript projects couldn't migrate to 11 without going
+  through 10 first. Fixes
```

Comment:
Won't all project always need to be migrated to 10 before the 11.0 migration occurs?
```suggestion
- Fixed an 11.0.0 regression where the Migration flow would error and hang for Typescript projects. Fixes
```

Reply:
Not if they're already on 10? But I can probably clarify this.

Reply:
Cypress 11 migration is the same as Cypress 10 migration, it's based on whether we detect a `cypress.json` file. People are able to migrate directly to 11 from 9 or lower.